### PR TITLE
[csc] Update CSC version from v1 to v2

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

**Comment by awgreene** [comment]
Please take a look @aravindhp @kevinrizza @galletti94

File: pkg/apis/operators/shared/phase_types.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package shared
```
Comment (aravindhp) [inline pkg/apis/operators/shared/phase_types.go]: I am assuming you did a `git mv` and git decided the file was new. Was that the case?

**Comment by aravindhp** [comment]
This should go in after #198

File: pkg/apis/operators/shared/phase_types.go (hunk 1 above)
Comment (awgreene) [reply pkg/apis/operators/shared/phase_types.go]: That is correct.

Comment (awgreene) [reply pkg/apis/operators/shared/phase_types.go]: It decided to create / delete the phase_type after adding the generated deepcopy functions.

**Comment by kevinrizza** [comment]
@awgreene @aravindhp Hey so I think I am slightly out of the loop here, but what is the plan here? Are we just planning on supporting both versions in a cluster? Would it not be better to just migrate any existing objects from V1 to V2 and then not support older versions?

**Comment by awgreene** [comment]
198 has closed, please review @aravindh @kevinrixza

**Pull request merged**